### PR TITLE
Feature: notifications screen and notification component design

### LINKED ISSUE #29: Feature: notifications screen design
#### Description

Because of many interactions in the app users need somewhere to be notified on actions that happened to their account. This notifications page will include all activities details, and announcements that are related to the logged in user.

#### Tasks

- [x] Design the interface
- [x] Import required icons
- [x] Extend the NavigationHeader component

#### Developer Notes

N/A

#### Related Issues

- N/A

#### Definition of Done

- All screens looks and behaves as the designs

### PR DESCRIPTION
**Pull request by fredshema**
#### What does this PR do?

This PR includes designs of the notifications screen and notification component

#### Description of Task to be completed?

- Design the notification screen with required header icons
- Design notification component

#### How should this be manually tested?

On the first page there is a link with text `Tabs`. It will redirect you to the home screen, click the bell icon and it will navigate you to the notifications screen.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Questions:

N/A

### CODE REVIEW

**Review by justinmihigo — APPROVED** [review]
what about the new button that shows the notification status

**Comment by fredshema** [comment]
> what about the new button that shows the notification status

Good catch let me add it👍🏽

**Review by twizerim — APPROVED** [review]

**Comment by mukunzidd** [comment]
@fredshema please fix the conflicts first.

File: components/notifications/notification.tsx
Comment (mukunzidd) [inline components/notifications/notification.tsx:1]: @fredshema Add a way for notifications to be marked as `new`. Look at the blue text on the far right of the latest notifications on the design.

Comment (mukunzidd) [reply components/notifications/notification.tsx:1]: The three dots on at the top right should have a button to clear the notifications. Just add it there. It doesn't have to be functional.

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "web": "expo start --web",
     "test": "jest",
     "build": "eas build -p android",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "start:wsl": "REACT_NATIVE_PACKAGER_HOSTNAME=$(netsh.exe interface ip show address 'Ethernet' | grep 'IP Address' | sed -r 's/^.*IP Address:\\W*//') expo start"
```
Comment (mukunzidd) [inline package.json]: Organize the scripts logically. For example all different kinds of start scripts should be together.

**Review by mukunzidd — CHANGES_REQUESTED** [review]
- Please address all the comments I made and make the notification body text more muted to improve the visual hierarchy. 
- Also leave less space between the notification details and the date so that they look like they belong together.

**Pull request merged**